workspace/viewer/PdfJsViewer.ts

```diff
@@ -573,12 +599,14 @@ export class PdfJsViewer {
     return result;
   }
 
-  private renderPage(pageNumber: number) {
+  // This method returns false if PageView was rendered. Returns true if it's still rendering.
```

Comment:
You don't seem to use the returned value in the calling code. What's the purpose of returning it then?

Reply:
Done: I've deleted return value.

It seemed to me quite useful for future to return if page was rendered, but I have to agree, that it's better to add anything when it's needed.